config/accounts.go

```diff
@@ -45,13 +45,16 @@ type CookieSync struct {
 type AccountCCPA struct {
 	Enabled            *bool              `mapstructure:"enabled" json:"enabled,omitempty"`
 	IntegrationEnabled AccountIntegration `mapstructure:"integration_enabled" json:"integration_enabled"`
+	ChannelEnabled     AccountIntegration `mapstructure:"channel_enabled" json:"channel_enabled"`
```

Comment:
It appears we are keeping both `IntegrationEnabled` and `ChannelEnabled` in the account config for now so that hosts have an opportunity to migrate from `integration_enabled` to `channel_enabled`, is that correct? I see this approach described in the associated issue and that PBS-Java has taken a similar approach where they eventually removed `integration_enabled` completely after the transition period which I assume we will also do in a future release.

Reply:
Yes this is correct!